backends-common/postgres/src/test/java/org/apache/james/backends/postgres/PostgresTableManagerTest.java

```diff
@@ -87,7 +87,7 @@ void initializeTableShouldSuccessWhenModuleHasSingleTable() {
             .createTableStep((dsl, tbn) -> dsl.createTable(tbn)
                 .column("colum1", SQLDataType.UUID.notNull())
                 .column("colum2", SQLDataType.INTEGER)
-                .column("colum3", SQLDataType.VARCHAR(255).notNull()));
+                .column("colum3", SQLDataType.VARCHAR(255).notNull())).noRLS();
```

Comment:
Return to the line for `.noRLS()` statement? Would be easier to read later

Idem for below